src/Xiaomi_Scale.py

```diff
@@ -176,16 +149,21 @@ def _publish(self, weight, unit, mitdatetime, hasImpedance, miimpedance):
         message += ',"TimeStamp":"' + mitdatetime + '"'
         message += '}'
         try:
-          self.mqtt_client.publish(MQTT_PREFIX + '/' + user + '/weight', message, qos=1, retain=True)
-          sys.stdout.write('Sent data to topic %s: %s' % (MQTT_PREFIX + '/' + user + '/weight', message + '\n'))
+            sys.stdout.write('Sent data to topic %s: %s' % (MQTT_PREFIX + '/' + user + '/weight', message + '\n'))
```

Comment:
stdout should go after attempting to publish, not before

Reply:
On that I disagree.

The way it was I couldn't see what message was failing, and it may be useful to debug.
Now logs says
- publishing blah blah blah

And if it fails
- publishing blah blah blah
- nope, it fails

Reply:
lol then we need to reword "sent" to "sending" then it makes sense and we then show the error if it failed😉